sklearn/linear_model/logistic.py

```diff
@@ -1345,8 +1346,8 @@ def fit(self, X, y):
         check_consistent_length(X, y)
 
         # init cross-validation generator
-        cv = check_cv(self.cv, X, y, classifier=True)
-        folds = list(cv)
+        cv = check_cv(self.cv, y, classifier=True)
```

Comment:
Do we need ``y``? I guess to be future proof in case we implement multi-label LR?